core/docker/compose.sh

```diff
@@ -1,7 +1,7 @@
 composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
 if [ "$1" == "up" ]; then
-    docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml  up -d
+    docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml up -d
 elif [ "$1" == "down" ]; then
     docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml stop
 elif [ "$1" == "destroy" ]; then
```

Comment:
@rcrichton Is there an easy way of removing the volumes associated with these containers on destroy?

I tried using the method described here:
https://docs.docker.com/compose/reference/rm/

But it didn't seem to work? Every time I run `docker-compose up` it created a new anonymous volume that is never cleaned up

Reply:
It might be easier to clean these up if we named the volumes and deleted them specifically...

Reply:
Ya sure can do that. I also see the `docker-compose rm` docs state: By default, anonymous volumes attached to containers are not removed. You can override this with -v

Reply:
I tried that. Didn't appear to work

Reply:
Ok, lets switch to named volume then, it's probably better. Are you ok to do this else I can look later today?

Reply:
I'll get on it :+1:

Reply:
Tried it again from a completely fresh start and it worked :| 
Think I must have had some other existing stuff mess with it :( 
But solved :D